runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/EngineConfiguration.java

```diff
@@ -82,7 +83,7 @@ public class EngineConfiguration extends Configuration
         ENGINE_CONFIG_URL = config.property(URL.class, "config.url", EngineConfiguration::configURL, "file:zilla.yaml");
         ENGINE_CONFIG_POLL_INTERVAL_SECONDS = config.property("config.poll.interval.seconds", 60);
         ENGINE_NAME = config.property("name", EngineConfiguration::defaultName);
-        ENGINE_DIRECTORY = config.property("directory", ".");
+        ENGINE_DIRECTORY = config.property("directory", EngineConfiguration::zillaDirectory);
```

Comment:
Following the same naming pattern for the methods, like `defaultName`, this method should be called `defaultDirectory`.